components/org.wso2.carbon.identity.scim.common/src/main/java/org/wso2/carbon/identity/scim/common/utils/SCIMCommonUtils.java

```diff
@@ -19,19 +19,37 @@
 package org.wso2.carbon.identity.scim.common.utils;
 
 import org.apache.commons.lang.StringUtils;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
 import org.wso2.carbon.CarbonConstants;
 import org.wso2.carbon.base.ServerConfiguration;
 import org.wso2.carbon.context.PrivilegedCarbonContext;
+import org.wso2.carbon.identity.core.util.IdentityTenantUtil;
+import org.wso2.carbon.identity.scim.common.internal.SCIMCommonComponentHolder;
+import org.wso2.carbon.stratos.common.util.ClaimsMgtUtil;
 import org.wso2.carbon.user.core.UserCoreConstants;
+import org.wso2.carbon.user.core.UserStoreManager;
+import org.wso2.charon.core.schema.SCIMConstants;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
 
 /**
  * This class is to be used as a Util class for SCIM common things.
  * TODO:rename class name.
  */
 public class SCIMCommonUtils {
 
+    private static final String SUPER_TENANT_DOMAIN = "carbon.super";
```

Comment:
Shall we use constants defined in org.wso2.carbon.utils.multitenancy.MultitenantConstants instead of defining it again here?

Reply:
Fixed with #73